File: php/Pc.php
<?php

include "Disk.php";
include "Processor.php";
include "Ram.php";


class Pc
{
    private $disk, $ram, $processor, $totalPrice;

    public function __construct()
    {
        $this->disk = new Disk();
        $this->ram = new Ram();
        $this->processor = new Processor();
        $this->totalPrice = 0;
    }

    public function setDisk($disk)
    {
        $this->disk = $disk;
    }

    public function setRam($ram)
    {
        $this->ram = $ram;
    }

    public function setProcessor($processor)
    {
        $this->processor = $processor;
    }

    public function setTotalPrice($totalPrice)
    {
        $this->totalPrice = $totalPrice;
    }

    public function getDisk()
    {
        return $this->disk;
    }

    public function getRam()
    {
        return $this->ram;
    }

    public function getProcessor()
    {
        return $this->processor;
    }

    public function getTotalPrice()
    {
        return $this->totalPrice;
    }
}

File: python/Pc.py
from dis import dis
import imp

from matplotlib.pyplot import cla
from Disk import Disk
from Ram import Ram
from Processor import Processor


class Pc():

    def __init__(self, disk=Disk(), ram=Ram(), processor=Processor(), totalPrice=0):
        self.__disk = disk
        self.__ram = ram
        self.__processor = processor
        self.__totalPrice = totalPrice

    def setDisk(self, disk):
        self.__disk = disk

    def setRam(self, ram):
        self.__ram = ram

    def setProcessor(self, processor):
        self.__processor = processor

    def setTotalPrice(self, totalPrice):
        self.__totalPrice = totalPrice

    def getDisk(self):  
        return self.__disk

    def getRam(self):
        return self.__ram

    def getProcessor(self):
        return self.__processor

    def getTotalPrice(self):
        return self.__totalPrice
